mentorApp/android/local.properties

```diff
@@ -0,0 +1,2 @@
+sdk.dir=/Users/Aysha/Library/Android/sdk
```

Comment:
This might conflict with everyone else's build as our sdks will not be in the same location.